modules/reactor/src/meshgenerators/PolygonConcentricCircleMeshGeneratorBase.C

```diff
@@ -342,25 +342,76 @@ PolygonConcentricCircleMeshGeneratorBase::PolygonConcentricCircleMeshGeneratorBa
     if (!_ring_block_ids.empty() &&
         _ring_block_ids.size() !=
             (_ring_intervals.size() + (unsigned int)(num_innermost_ring_layers != 1)))
```

Comment:
I dont understand why we need the extra block id if _quad_center_elements is true ?

Reply:
If quad elements is true, the user can choose to provide it or not. But if he doesn't the one gets duplicated.

Reply:
ok seems fine but it s an implementation detail. focus on making the error message clear, then you can mention that as a note at the end.